### DISCUSSION TITLE
[Question] Should doodba deployments pin a specific version or always be up to date?

### DISCUSSION THREAD

**Discussion by obayit** [Tecnativa/doodba#386] [Q&A] [answered]
If I am using doodba-copier-template for production, should I 1) pin a specific version and only use that, with also a specific `tag` of doodba docker image? 
Or should I 2) always keep my projects up to date, and nothing will break?

My guess is to use a specific version of the template and the docker image, and then updating them monthly for example. But I still didn't try that, and don't know if it is possible.
The second option - keeping projects up to date (updating the copier template from the official `repo`, and from the last commit) and pulling new images all the time - sounds good but it needs careful following of new breaking changes.

Is there another option?

**Comment by yajo** [comment, marked as answer]
Well, you already said both approaches and the pros and cons. All I can add is that you have to evaluate the pros and cons, and decide which one is better for you.

The project itself is designed for constantly upgrading to the latest version, but that doesn't mean you can't pin things downstream if you prefer that.
